### ISSUE TITLE
Allow admins to add accessibility statement

### ISSUE THREAD

**Issue by fisharebest** [IET-OU/cloudengine#314]
At the moment the only way to add an accessibility statement is as a page. There should be a way to put in a link to an accessibility statement (which could still be a page) in the footer and to ensure that it is one of the first things that a screenreader will see. 

---
- Bitbucket: https://bitbucket.org/cloudengine/cloudengine/issue/314
- Originally Reported By: [Juliette Culver](http://bitbucket.org/julietteculver)
- Originally Created At: 2013-01-10T17:41:35.583
